diff --git a/configure.ac b/configure.ac
index c680bda7..72f8927e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -763,7 +763,6 @@ GCC_WARNINGS="-Wall -Wstrict-prototypes -Wformat-security -Wshadow"
 AS_IF([test "x${gcc_wextra}" = "xyes"], [
   GCC_WARNINGS="${GCC_WARNINGS} -Wextra -Wno-unused-parameter"
   GCC_WARNINGS="${GCC_WARNINGS} -Wbad-function-cast"
-  GCC_WARNINGS="${GCC_WARNINGS} -Wcast-align"
   GCC_WARNINGS="${GCC_WARNINGS} -Wredundant-decls"
 ])
 AS_IF([test "x${gcc_werror}" = "xyes"], [
